glade_forms/README.md

```diff
@@ -339,6 +306,27 @@ onChange: (info, dependencies) {
 
 ![two-way-inputs-example](https://raw.githubusercontent.com/netglade/glade_forms/main/glade_forms/doc/two-way-dependencies.gif)
 
+Second approach is to use `dependencies` and `onDependencyChange` callback and react when different input was changed.
+
+In this example, when age-input updats its value (dependency), checkbox's value (vipInput) is updated.
+
+```dart
+ vipInput = GladeInput.create(
+    inputKey: 'vip-input',
+    dependencies: () => [ageInput],
+    onChange: (info) {
+      if (info.value && ageInput.value < 18) {
+        ageInput.value = 18;
+      }
+    },
+    onDependencyChange: (key) {
```

Comment:
Just thinking: if you have multiple dependencies (lets say 3) and you do group edit containing multiple/all of them, should onDependencyChange be called multiple times or only once and "key" should be "keys" then?

Reply:
Not possible - we are listening in input over individual ChangeNotifiers. Input doesn't know about "groupEdit". But yeah I was thinking about same... 

We could probably move this logic up to GladeModel somehow

Reply:
Refactored - now glade model handle notifications